pcsx2/CDVD/CsoFileReader.h

```diff
@@ -15,9 +15,22 @@
 
 #pragma once
 
+// Based on testing, the overhead of using this cache is high.
+//
+// The test was done with CSO files using a block size of 16KB.
+// Cache hit rates were observed in the range of 25%.
+// Cache overhead added 35% to the overall read time.
```

Comment:
Interesting, but not extremely surprising. The cache is searched linearly without any hash, and the relatively small chunks size and relatively big cache ends up with a long list to search per access. Thanks for taking the time to measure.